frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -1019,4 +1019,13 @@ function ReaderPaging:onGotoPrevChapter()
     return true
 end
 
+function ReaderPaging:onToggleReadingOrder()
+    self.inverse_reading_order = not self.inverse_reading_order
+    self:setupTapTouchZones()
+    UIManager:show(Notification:new{
+        text = self.inverse_reading_order and _("RTL page turning.") or _("LTR page turning."),
+        timeout = 2.5,
+    })
```

Comment:
Speaking of bubbling, this should return true to stop propagation.

Reply:
will fix